### ISSUE TITLE
Some cards are included in different quantities based on the number of players

### ISSUE THREAD

**Issue by jaypatel1227** [evanofslack/pyminion#97]
One example is Gardens [Source[(http://wiki.dominionstrategy.com/index.php/Gardens#Official_FAQ). Should this be considered or not?

**Comment by evanofslack** [comment]
Good catch, all victory cards should be adjusted based on number of players.

**Comment by jbw3** [comment]
This issue should be fixed with #109.
